CONTRIBUTING.md

```diff
@@ -83,6 +69,18 @@ Here are a few things you can do that will increase the likelihood of your pull
 
 Work in Progress pull requests are also welcome to get feedback early on, or if there is something blocked you.
 
+## Releasing
+
+In order to release a new version, you need to compile the package using `npm run build`. This will output a `dist/index.js` file.
+Then, perform the following actions:
+
+- `git add dist/index.js`
+- `git commit -m "Release x.x.x"`
```

Comment:
in future, we should consider integrating [committzen](https://www.google.com/search?channel=fs&client=ubuntu&q=committizen)

Reply:
Great idea. Want to add an issue?